admin/package.json

```diff
@@ -4,19 +4,18 @@
   "private": false,
   "license": "GPL-3.0",
   "scripts": {
-    "coverage": "nyc -r text -r html ./run-jasmine.js",
     "lint": "standard",
     "migrate-sql": "node ./data/sql/migrate-sql.js",
     "seed-sql": "node ./data/sql/seed-sql.js",
     "migrate-and-seed-sql": "yarn migrate-sql && yarn seed-sql",
     "postinstall": "gulp build",
     "sasswatch": "gulp watch",
-    "sql-integration-test": "nyc -r html ./tests-integration/run-jasmine-integration.js",
     "start": "node ./server.js",
     "startd": "yarn migrate-sql && node ./server.js",
-    "test": "yarn lint && yarn run nyc -r html ./run-jasmine.js",
-    "client-test": "karma start karma.conf.js",
-    "testDebug": "node $NODE_DEBUG_OPTION ./node_modules/.bin/jasmine",
+    "test": "yarn lint && yarn jest",
+    "test:watch": "yarn jest --watch",
+    "test:integration": "yarn jest --config ./tests-integration/jest.integration.config.js",
+    "test:frontend": "karma start karma.conf.js",
```

Comment:
should we change frontend to run on jest as well?

Reply:
Karma supports auto reload i believe, and any conversion to jest would need to be in a separate ticket